include/power/power.h

```diff
@@ -53,24 +53,24 @@ struct pm_notifier {
 	 * Application defined function for doing any target specific operations
 	 * for power state entry.
 	 */
-	void (*state_entry)(enum pm_state state);
+	void (*state_entry)(struct pm_state state);
```

Comment:
Various of these changes ought to be in the previous commit for bisectability, no?